### PR TITLE
create initial template for Rails 5 apps

### PR DESCRIPTION
**Pull request by ericcf**

### CODE REVIEW

File: rails5.rb [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+inside "config" do
+  insert_into_file "application.rb",
+                   "\n    config.time_zone = \"#{timezone}\"\n",
+                   after: "class Application < Rails::Application"
+end
+
+# Update with code conventions
+copy_file ".rubocop.yml"
+run "#{rvm_do} rubocop -a"
+inside "app" do
```
Comment (michaelwehrley) [inline rails5.rb]: 👍

File: rails5.rb [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+  insert_into_file "mailers/application_mailer.rb",
+                   "# Mailer superclass.\n",
+                   before: "class ApplicationMailer < ActionMailer::Base"
+  insert_into_file "models/application_record.rb",
+                   "# Persisted model superclass.\n",
+                   before: "class ApplicationRecord < ActiveRecord::Base"
+end
+
+# Prepare database
+run "#{rvm_do} rails db:create db:migrate"
```
Comment (michaelwehrley) [inline rails5.rb]: Did you also want to set up test database via `RAILS_ENV=test rails db:migrate`?

File: rails_root/app/views/devise/confirmations/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<h2><%= t('.resend_confirmation_instructions') %></h2>
+
+<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+  <%= devise_error_messages! %>
+
+  <div class="form-group row">
```
Comment (michaelwehrley) [inline rails_root/app/views/devise/confirmations/new.html.erb]: If we are adding classes here, I would instead follow bootstrap [convention](http://v4-alpha.getbootstrap.com/components/forms/#form-groups) and remove the `br` and `div` tags and `row` class:
```html
<div class="form-group">
  <%= f.label :email %>
  <%= f.email_field :email,
                    autofocus: true,
                    value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email),
                    class: "form-control"
  %>
</div>
```

File: rails_root/app/views/devise/mailer/confirmation_instructions.html.inky [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<container>
```
Comment (michaelwehrley) [inline rails_root/app/views/devise/mailer/confirmation_instructions.html.inky]: Thanks for including inky. 😈

File: rails_root/app/views/devise/registrations/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,22 @@
+<p class="lead"><%= t('.ready_to_quit') %></p>
```
Comment (michaelwehrley) [inline rails_root/app/views/devise/registrations/new.html.erb]: These translation helpers are from another app; I would remove.

File: rails5.rb (hunk 2 above)
Comment (ericcf) [inline rails5.rb]: Yeah, I didn't realize that `rails db:create` creates both test and development databases, but migrate only migrates the current environment...

File: rails_root/app/views/devise/registrations/new.html.erb (hunk 5 above)
Comment (ericcf) [inline rails_root/app/views/devise/registrations/new.html.erb]: Oops

File: rails_root/app/views/layouts/bootstrap_application.html.erb [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  </head>
+
+  <body>
+    <div class="container">
+      <span id="flash">
+        <%= render "/shared/flash", locals: { notice: notice, alert: alert } %>
+      </span>
+
+      <div class="row">
+				<%= yield %>
```
Comment (michaelwehrley) [inline rails_root/app/views/layouts/bootstrap_application.html.erb]: We can fix the spacing in another PR.

Comment (ericcf) [inline rails_root/app/views/layouts/bootstrap_application.html.erb]: Darn tabs!

**Pull request merged**